src/api/python/z3compat/z3.py

```diff
@@ -0,0 +1,255 @@
+############################################
+# Copyright (c) 2021 The CVC5 Developers
+#               2012 The Microsoft Corporation
+#
+# CVC5's Z3-based Python interface
```

Comment:
Nitpicky, but I wonder if "Z3-based" is the right terminology here? It uses some of the pyz3 code, but otherwise doesn't actually _use_ Z3 at all right?

Reply:
it is a z3 compatibility layer on top of cvc5, I wouldn't call it z3-based.

Reply:
Done in f774ebf5ac